homework09/homework09/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace homework09
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var song = new Song();
+            Console.WriteLine(song.GetSongData(song)); 
+            Console.Read();
+        }
+    }
+
+    class Song
+    {
+        private string _title;
+        private int _minutes;
+        private string _author;
+        private int _year;
+        private Genre _genre;
+
+        
+
+        public dynamic GetSongData(Song song)
```

Comment:
I see that song parameter is not really used into this method. 
I would suggest to move the following piece of code to the Main method and after initialize Song object and put this object in `GetSongData` as a parameter:
![image](https://user-images.githubusercontent.com/3023032/56132137-e3e76480-5f91-11e9-8a33-e0ccb89dfb71.png)